### ISSUE TITLE
Rephrase the first paragraph of README to be more neutral

### ISSUE THREAD

**Issue by ChALkeR** [qmlweb/qmlweb#91]
This text does not look neutral to me and could be viewed as being offensive by someone:
> CSS and HTML are boring and lame. And they suck at designing cool, interactive interfaces. Qt came up with a much better answer for its renowned framework: QML, a declarative language perfect for designing UIs (and much more). Here's a sample of how QML looks like

We should better rephrase it to be more neutral. I'm not a native English speaker, and I would be glad if someone else filed a pull request to fix this.

**Comment by Plaristote** [comment]
Truthfully, when I wrote this, I wasn't expecting anyone to even take a look at my repository. I was writing this mostly for myself.
I agree that it should change. Although I *do* believe CSS and HTML are boring and lame !

We could link to the Wikipedia page:
https://en.wikipedia.org/wiki/QML

Also, perhaps either imitate or link to the QML introduction from the official docs:
http://doc.qt.io/qt-4.8/qdeclarativeintroduction.html

It's hard to sell QML to web developers... I think this documentation page is alright, but it present QML as something that only does interfaces... which is something HTML/CSS/JS already do pretty well.

The property binding, the signals, the contexts are what make QML great for a web-page as well.
QMLWeb really is just something that would take the place of Backbone and React on your application (or anything that plays that part, be it Angular or something else). We could present it as an integrated, coherent and somewhat standardized way of doing what is usually done with a myriad of javascript libraries.

**Comment by ChALkeR** [comment]
Closed in 26cd14f.

See #266 for ideas for further improvements.
